fairing/preprocessors/function.py

```diff
@@ -0,0 +1,49 @@
+import fairing
+import glob
+import os
+from pathlib import Path
+
+from fairing.constants import constants
+from .base import BasePreProcessor
+
+FUNCTION_SHIM = 'function_shim.py'
+
+
+class FunctionPreProcessor(BasePreProcessor):
+    """
+    FunctionPreProcessor preprocesses a single function.
+    It sets as the command a function_shim that calls the function directly.
+
+    args: function_name - the name of the function to be called
+    """
+    def __init__(self,
+                 function_name,
+                 executable=None,
```

Comment:
there is no doc strings in BasePreProcessor so I am not able to get clear idea on what executable variable is and why it is not passed to super().\_\_init\_\_ fn.

Reply:
good point - removed it and added a docstring in the base preprocessor